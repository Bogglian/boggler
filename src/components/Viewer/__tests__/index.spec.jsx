import React from 'react';
import { shallow } from 'enzyme';
import { Viewer } from '..';

describe('Viewer Component', () => {
  describe('user possibly create a Viewer component.', () => {
    it('should render a Viewer', () => {
      const component = shallow(<Viewer />);

      expect(component).toMatchSnapshot();
    });
  });
});
