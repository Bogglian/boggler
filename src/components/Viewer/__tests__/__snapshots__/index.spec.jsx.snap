// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Viewer Component user possibly create a Viewer component. should render a Viewer 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Viewer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "astPlugins": Array [],
      "escapeHtml": true,
      "plugins": Array [],
      "rawSourcePos": false,
      "renderers": Object {},
      "skipHtml": true,
      "source": undefined,
      "sourcePos": false,
      "transformLinkUri": [Function],
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "astPlugins": Array [],
        "escapeHtml": true,
        "plugins": Array [],
        "rawSourcePos": false,
        "renderers": Object {},
        "skipHtml": true,
        "source": undefined,
        "sourcePos": false,
        "transformLinkUri": [Function],
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
